### DISCUSSION TITLE
Imputation

### DISCUSSION THREAD

**Discussion by beantkapoor786** [maize-genetics/phg_v2#188] [Q&A] [answered]
How does PHG_v2 handle the imputation of a sample if one or few of its ancestral genomes are not included in the pangenome itself? Does it still attempt to impute the 'most likely path' or it does not impute that region of the genome?

**Comment by zrm22** [comment, marked as answer]
Hello,

If a sample has ancestral genomes that are not included in the pangenome the PHGv2 by default will attempt to impute the regions which it does not have read support for.  

You can set --min-reads  to 1 or higher to prevent this during the find-paths step.  But you will likely have gaps(reference ranges where a Haplotype is not called) in the path wherever you have reference ranges which do not have a single read hitting a Haplotype within that range.

**Reply by beantkapoor786** [reply]
Okay, thank you for the quick response!
